src/main/java/core/basesyntax/service/date/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.date;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private Map<String, Integer> storage;
+
+    public ReportGeneratorImpl(Map<String, Integer> storage) {
+        this.storage = storage;
+    }
```

Comment:
Avoid using the absolute path to a resource. Everyone who pulls your project should be able to run it. Please provide the relative path to a resource instead.